src/index.css

```diff
@@ -61,6 +61,15 @@
   src: url('./fonts/open-sans-v27-latin-800italic.woff2') format('woff2');
 }
 
+
 .awsui-dark-mode {
   color-scheme: dark;
 }
+
+body{
+  color: #000716;
```

Comment:
These colour values differ between VR and Classic, so hardcoding them here will lead to colour differences for applications using Classic. I think we need to use design tokens for this. Since they're not available in global-styles, maybe there is a way we can put this style in the components package instead?

Reply:
I think for open source we only need to take care of VR? Added different value for VR and classic in CR-86495135.

Reply:
I also have concerns about this approach. Not just about using hard-coded colors, but also about adding colors to `body`.

I wonder if a Spinner sitting on just a `body` background is a valid use-case...

Reply:
Will close this PR and work on Applayout scope.